docs/001-devices.apib

```diff
@@ -243,3 +243,67 @@ Provides a listing of available services plans available to provision your devic
                 ...
               ]
             }
+
+## Device IP Address [/devices/{id}/ips]
+Assign an IP to a device, it must be reserved to my project first.
```

Comment:
I think the wording here is a little funny. Perhaps something like this instead:

> Assign an IP address to a device. The IP address must be in one of the IP ranges assigned to the device's project.